src/renderer/components/composer/Composer.tsx

```diff
@@ -305,7 +305,7 @@ const Composer = forwardRef<
       </div>
     )
   } else if (isDisabled) {
-    return <DisabledMessageInput ref={ref} reason={disabledReason} />
```

Comment:
The `ref` is used by `MessageListAndComposer` to automatically give focus on the input area. Since none input exists at this point (read-only group) we also don't need that reference

Reply:
good catch